src/test/java/com/artipie/http/VersionSliceTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie.http;
+
+import com.artipie.ArtipieProperties;
+import com.artipie.asto.test.TestResource;
+import com.artipie.http.hm.RsHasBody;
+import com.artipie.http.hm.RsHasStatus;
+import com.artipie.http.hm.SliceHasResponse;
+import com.artipie.http.rq.RequestLine;
+import com.artipie.http.rq.RqMethod;
+import com.artipie.http.rs.RsStatus;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Properties;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for {@link VersionSlice}.
+ * @since 0.21
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ */
+final class VersionSliceTest {
+    @Test
+    void returnVersionOfApplication() throws IOException {
+        final Properties properties = new Properties();
+        properties.load(new TestResource("artipie.properties").asInputStream());
+        MatcherAssert.assertThat(
+            new VersionSlice(new ArtipieProperties()),
+            new SliceHasResponse(
+                Matchers.allOf(
+                    new RsHasStatus(RsStatus.OK),
+                    new RsHasBody(
+                        String.format(
+                            "[{\"version\":\"%s\"}]",
```

Comment:
It's not always valid to verify json with a string (since it can be valid in multiple formats, e.g. with white-spaces, newlines, etc). So here [json matcher for body](https://github.com/artipie/http/blob/master/src/main/java/com/artipie/http/hm/IsJson.java) could be used: `new RsHasBody(new IsJson(new JsonHas("version", new JsonValueIs("1.0")))`